### ISSUE TITLE
RemoteApp does not show correctly UAC

### ISSUE THREAD

**Issue by IridiumXOR** [FreeRDP/FreeRDP#10495]
If you use xfreerdp to connect to a RemoteApp and the remote app call the UAC screen (the request for admin privileges), xfreerdp open a huge black window, impossible to close and to interact with it and no UAC is shown.

**Comment by mattcbro** [comment]
Yup.  The /app:program:  option does not appear to be working with a linux mint host and a windows 11 virtual machine.
